### PR TITLE
BSIP 87: Force Settlement Fee Ratio

### PR DESCRIPTION
**Pull request by bitcrab**

### CODE REVIEW

**Comment by abitmore** [comment]
The commit history is a bit messy.. @bitcrab could you please remove the unnecessary changes about BSIP81? If you don't know how to rebase/squash/force push, perhaps create a new pull request. Thanks.

File: bsip-0087.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    Type: Protocol
+    Created: 2020-02-23
+    Discussion: https://github.com/bitshares/bsips/issues/260
+    Worker: TBD
+
+# Abstract
+This BSIP provide a solution to charge fee from force settlement.
+
+# Motivation
+Force settlement is an important part in smartcoin design, it provides the power to smartcoin owner to ask for collaterals with reference to feed price, and then guarantees the peg of smartcoin.
```
Comment (abitmore) [inline bsip-0087.md]: "smartcoin owner" is used several times in the document, but meaning different group of people. I think it should be "smartcoin holders" here.

File: bsip-0087.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+# Rationale
+Force settlement fee can be seen as to pay part of the cost for smartcoin supply, stabilization and also the liquidation of collaterals, it is irrelevant to market fee sharing.
+To ensure the debt positions can always be closed with suffcient smartcoin, it is more feasible to cut off part of the traded collaterals as fee, instead of smartcoin.
+
+# Specification
+Add one new parameter Force Settlement Fee Ratio(FSFR) for each smartcoin, which is controlled by the smartcoin owner.
+`Force settlement order price = settlement price*(1+FSO)/(1-FSFR)`
+
+Here settlement price is a new introduced parameter which is defined in BSIP71, settlement price = feed price when there is no bad debt.
```
Comment (abitmore) [inline bsip-0087.md]: BSIP 71 is not finalized yet.

**Comment by sschiessl-bcp** [comment]
> The commit history is a bit messy.. @bitcrab could you please remove the unnecessary changes about BSIP81? If you don't know how to rebase/squash/force push, perhaps create a new pull request. Thanks.

As an admin of the bsip repository you can commit to the branch of incoming PR's if the requester did not deactivate it specifically.

**Comment by bitcrab** [comment]
a new PR #265 is created to replace this one, please check.

**Pull request closed without merging**